tests/gradients/parameter_shift/test_parameter_shift_shot_vec.py

```diff
@@ -2109,7 +2109,10 @@ def test_trainable_coeffs(self, broadcast, tol):
 
         res = dev.execute([tape])
         expected = -c * np.sin(x) * np.sin(y) + np.cos(x) * (a + b * np.sin(y))
-        assert np.allclose(res, expected, atol=tol, rtol=0)
+        # This test is mostly for the shot vector pipeline check
+        # therefore only check a rough accuracy
+        assert qml.math.shape(res)[1] == qml.math.shape(shot_vec)[0]
+        assert np.allclose(res, expected, atol=2 * tol, rtol=0)
```

Comment:
```suggestion
```
Another way to eternally resolve this issue; after all, this test suite is not supposed to take care of accuracy (there should have been separate, specialized unit test suite to look into the accuracy of independent finite-shot execution accuracy). If this is approved by everyone, then we also need to delelete the `tol` from testing arg.